resources/scss/ceres/views/PageDesign/_footer.scss

```diff
@@ -1,58 +1,34 @@
 .to-top {
-    display: block;
-    width: 100%;
-    padding: .5rem;
-    text-align: center;
-    text-decoration: none !important;
-    cursor: pointer;
     background-color: $gray-700;
 
     &:hover {
         background-color: $gray-900;
     }
     i {
-        padding: .5em 0;
         font-size: 1.5em;
-        color: white;
     }
 }
 
 .back-to-top-center {
     position: fixed;
+    right: 0;
     bottom: -17px;
-    left: 50%;
+    left: 0;
     z-index: 1000;
     display: none;
     width: 94px;
     height: 50px;
-    padding: 20px;
-    margin-left: -47px;
-    overflow: hidden;
-    text-align: center;
-    text-decoration: none;
-    cursor: pointer;
     background: #fff;
-    border: 1px solid $gray-900;
-    border-top-left-radius: 25px;
-    border-top-right-radius: 25px;
     opacity: .5;
-    -webkit-transition: opacity .3s;
-    transition: opacity .3s;
+    transition: opacity .3s; // stylelint-disable-line
```

Comment:
Warum nicht das transition-mixin? stylelint-disable wenn nur in Kombination mit einer konkreten Regel verwenden.

Reply:
durch beta5 merge behoben